frontends/web/test/i18n/config.test.tsx

```diff
@@ -76,7 +76,7 @@ describe('language detector', () => {
     it('uses native-locale if userLanguage is null', done => {
         (apiGet as jest.Mock).mockImplementation(endpoint => {
             switch (endpoint) {
-                case 'config': { return Promise.resolve({frontend: {userLanguage: null}}); }
+                case 'config': { return Promise.resolve({backend: {userLanguage: ""}}); }
```

Comment:
So it used to be stored as null in config.json, how can it be empty string?

Name of the test is 'uses native-locale if userLanguage is null'

Does `migrateUserLanguage` turn null to empty string?

Reply:
Ok backend conf is now an empty stirng if the lang == native lang 👍 

but the frontend conf is still there, remove?

![Screen Shot 2021-01-20 at 10 15 56 AM](https://user-images.githubusercontent.com/546900/105153420-b5080600-5b08-11eb-9f7a-474fbce2ebc3.png)

Reply:
Add this comment. Makes sense?

![image](https://user-images.githubusercontent.com/25405/105200499-2fa04800-5b40-11eb-98fc-d63000670979.png)

Reply:
thank you :)